web/src/features/panels/search-panel/getSearchData.ts

```diff
@@ -0,0 +1,41 @@
+import { t } from 'i18next';
+import { GridState } from 'types';
+
+import { ZoneRowType } from './ZoneList';
+
+export const getAllZones = () => {
+  // Get all zone data directly from translations
+  const zoneData = t('zoneShortName', { returnObjects: true }) as Record<
+    string,
+    ZoneRowType
+  >;
+
+  return zoneData;
+};
+
+export const getFilteredList = (
+  searchTerm: string,
+  zoneData: Record<string, ZoneRowType>
+): ZoneRowType[] => {
+  if (!searchTerm) {
+    return [];
+  }
+
+  // Filter zones based on search term
+  const filtered = Object.entries(zoneData).filter(([zoneKey, zone]) => {
+    const searchLower = searchTerm.toLowerCase();
+    return (
+      zoneKey.toString().toLowerCase().includes(searchLower) ||
```

Comment:
why is this not using zoneID (which is the zoneKey) as before?

Would be easier to deal with one object if we want to add fuzzy search using https://github.com/farzher/fuzzysort for example.

Reply:
FYI that library only seems to support English so that might be a dealbreaker.

Reply:
Requires more work and reprocessing the translations object to add it in. Added a comment though that we can change this if we need it in the future :)